README.md

```diff
@@ -75,7 +75,7 @@ These sections describe requirements for using this module.
 
 The following dependencies must be available:
 
-- [Terraform][terraform] v0.11
+- [Terraform][terraform] v0.12
```

Comment:
Thoughts on providing users of this module context on the last known-working 0.11.x commit in the README, so users that haven't upgraded aren't out of luck?

Reply:
For now, the task is resolved for and module updated for usage with terraform-0.12 with no backward compatibility support, as far I see no way to support it because terraform-0.12 haven't full backward compatibility with previous versions of terraform. I assume support of TF0.11 should be in a separated TF0.11 branch.